Submit notebook as pipeline fails

**Describe the issue**
See error message

**To Reproduce**
Steps to reproduce the behavior:
1. Create a simple notebook
2. Configure a KFP runtime environment
3. Submit notebook for execution in that env
4. See error. Note the reference to Airflow even though KFP was selected as runtime

**Screenshots or log output**
If applicable, add screenshots or log output to help explain your problem.
<details><summary>Log Output</summary>
<pre>
[I 2021-02-17 15:26:56.219 ServerApp] airflow 'Untitled-0217152656' - Submitting pipeline 
[I 2021-02-17 15:26:56.220 ServerApp] Creating pipeline definition as a .py file
[I 2021-02-17 15:26:56.232 ServerApp] airflow 'Untitled-0217152656' - processing pipeline dependencies to: http://cloning1.fyre.ibm.com:31671 bucket: pipeline-artifacts folder: Untitled-0217152656 
[I 2021-02-17 15:26:56.251 ServerApp] airflow 'Untitled-0217152656':'Untitled' - processing operation dependencies for id: 46d4d67f-3241-4ae6-87a3-1636f02cfe9b 
[I 2021-02-17 15:26:56.254 ServerApp] airflow 'Untitled-0217152656':'Untitled' - generated dependency archive: /var/folders/h8/l3qh8v192rv_vvggc6wyphzc0000gn/T/elyra/Untitled-46d4d67f-3241-4ae6-87a3-1636f02cfe9b.tar.gz (0.003 secs)
[I 2021-02-17 15:26:56.913 ServerApp] airflow 'Untitled-0217152656':'Untitled' - uploaded dependency archive to: Untitled-0217152656/Untitled-46d4d67f-3241-4ae6-87a3-1636f02cfe9b.tar.gz (0.173 secs)
[I 2021-02-17 15:26:56.913 ServerApp] airflow 'Untitled-0217152656' - pipeline dependencies processed (0.681 secs)
[E 2021-02-17 15:26:56.965 ServerApp] Uncaught exception POST /elyra/pipeline/schedule?1613604416188 (127.0.0.1)
    HTTPServerRequest(protocol='http', host='127.0.0.1:8888', method='POST', uri='/elyra/pipeline/schedule?1613604416188', version='HTTP/1.1', remote_ip='127.0.0.1')
    Traceback (most recent call last):
      File "/opt/anaconda3/envs/elyra_dev/lib/python3.7/site-packages/tornado/web.py", line 1704, in _execute
        result = await result
      File "/opt/anaconda3/envs/elyra_dev/lib/python3.7/site-packages/elyra/pipeline/handlers.py", line 89, in post
        response = await PipelineProcessorManager.instance().process(pipeline)
      File "/opt/anaconda3/envs/elyra_dev/lib/python3.7/site-packages/elyra/pipeline/processor.py", line 75, in process
        res = await asyncio.get_event_loop().run_in_executor(None, processor.process, pipeline)
      File "/opt/anaconda3/envs/elyra_dev/lib/python3.7/concurrent/futures/thread.py", line 57, in run
        result = self.fn(*self.args, **self.kwargs)
      File "/opt/anaconda3/envs/elyra_dev/lib/python3.7/site-packages/elyra/pipeline/processor_airflow.py", line 79, in process
        repo = github_client.get_repo(github_repo)
      File "/opt/anaconda3/envs/elyra_dev/lib/python3.7/site-packages/github/MainClass.py", line 338, in get_repo
        assert isinstance(full_name_or_id, (str, int)), full_name_or_id
    AssertionError: None
</pre>
</details>

**Expected behavior**
A clear and concise description of what you expected to happen.

**Deployment information**
Describe what you've deployed and how:
 - Elyra version: 2.0.x
 - Operating system: macos
 - Installation source:  from source
 - Deployment type: local installation

**Pipeline runtime environment**
If the issue is related to pipeline execution, identify the environment where the pipeline is executed
 - Kubeflow Pipelines (provide version number, whether multi-user auth enabled)
 - Apache Airflow (provide version number)
 
**Runtime configuration settings**
Not relevant in this scenario

--- a/packages/pipeline-editor/src/SubmitNotebookButtonExtension.tsx
+++ b/packages/pipeline-editor/src/SubmitNotebookButtonExtension.tsx
@@ -71,6 +71,7 @@ export class SubmitNotebookButtonExtension
     }
 
     const {
+      runtime_platform,
       runtime_config,
       framework,
       dependency_include,
@@ -78,9 +79,12 @@ export class SubmitNotebookButtonExtension
       ...envObject
     } = dialogResult.value;
 
+    console.log(dialogResult.value);
+
     // prepare notebook submission details
     const pipeline = Utils.generateNotebookPipeline(
       this.panel.context.path,
+      runtime_platform,
       runtime_config,
       framework,
       dependency_include ? dependencies : undefined,

--- a/packages/pipeline-editor/src/SubmitNotebookButtonExtension.tsx
+++ b/packages/pipeline-editor/src/SubmitNotebookButtonExtension.tsx
@@ -79,8 +79,6 @@ export class SubmitNotebookButtonExtension
       ...envObject
     } = dialogResult.value;
 
-    console.log(dialogResult.value);
-
     // prepare notebook submission details
     const pipeline = Utils.generateNotebookPipeline(
       this.panel.context.path,

--- a/packages/pipeline-editor/src/utils.ts
+++ b/packages/pipeline-editor/src/utils.ts
@@ -34,6 +34,7 @@ export default class Utils {
    */
   static generateNotebookPipeline(
     filename: string,
+    runtime_platform: string,
     runtime_config: string,
     runtimeImage: string,
     dependencies: string[],
@@ -59,7 +60,7 @@ export default class Utils {
     template.pipelines[0].nodes[0].app_data.dependencies = dependencies;
 
     template.pipelines[0].app_data.name = artifactName;
-    template.pipelines[0].app_data.runtime = 'airflow';
+    template.pipelines[0].app_data.runtime = runtime_platform;
     template.pipelines[0].app_data['runtime-config'] = runtime_config;
     template.pipelines[0].app_data.version = PIPELINE_CURRENT_VERSION;
 

